services/ui-src/src/components/forms/FormLayoutElements.tsx

```diff
@@ -12,15 +13,20 @@ export const SectionHeader = ({
     label: {
       marginTop: "2rem",
     },
+    hr: {
+      margin: divider === "top" ? "3rem 0" : "0",
+    },
   };
   return (
     <Box sx={sx} {...props}>
+      {divider === "top" && <hr></hr>}
       <Heading size="sm" sx={sx.label}>
         {label || ""}
       </Heading>
       <Heading size="md" sx={sx.h3}>
         {content}
       </Heading>
+      {divider === "bottom" && <hr></hr>}
```

Comment:
I only saw `"divider": "top"` in the sar. Do we need the bottom one if we are not using it?